api/lib/domain/models/JuryCertification.js

```diff
@@ -2,6 +2,36 @@ import { CompetenceMark } from './CompetenceMark.js';
 import { JuryComment, JuryCommentContexts } from '../../../src/certification/shared/domain/models/JuryComment.js';
 
 class JuryCertification {
+  /**
+   * @param {Object} props
+   * @param {number} props.certificationCourseId
+   * @param {number} props.sessionId
+   * @param {number} props.userId
+   * @param {number} props.assessmentId
+   * @param {string} props.firstName
+   * @param {string} props.lastName
+   * @param {string} props.birthplace
+   * @param {string} props.birthINSEECode
+   * @param {string} props.birthCountry
+   * @param {string} props.birthPostalCode
+   * @param {Date} props.createdAt
+   * @param {Date} props.completedAt
+   * @param {string} props.status
+   * @param {boolean} props.isCancelled
+   * @param {boolean} props.isPublished
+   * @param {boolean} props.isRejectedForFraud
+   * @param {number} props.juryId
+   * @param {number} props.pixScore
+   * @param {Array<CompetenceMark>} props.competenceMarks
+   * @param {JuryComment} props.commentForCandidate
+   * @param {JuryComment} props.commentForOrganization
+   * @param {string} props.commentByJury
+   * @param {string} props.commentByAutoJury
```

Comment:
Cette ligne apparaît en warning dans mon IDE : `Parameter commentByAutoJury described in JSDoc does not appear in function signature`.

Effectivement, la propriété `commentByAutoJury` n'est pas utilisée dans le constructeur (seulement destructurée depuis le `juryCertificationDTO` dans la méthode `from`. On peut l'enlever de la JS Doc ? :)

Reply:
Done dernier commit de refacto